modes/mu4e/evil-collection-mu4e.el

```diff
@@ -307,8 +312,9 @@ with older release versions of `mu4e.'"
   "Define the evil-mu4e Misc region."
   (concat
    (evil-collection-mu4e--main-action-str "\t* [;]Switch focus\n" 'mu4e-context-switch)
-   (evil-collection-mu4e--main-action-str "\t* [u]pdate email & database (Alternatively: gr)\n"
-                         'mu4e-update-mail-and-index)
+   (evil-collection-mu4e--main-action-str "\t* [u]pdate email & database"
+                                          'mu4e-update-mail-and-index)
+   (evil-collection-mu4e--main-action-str " (Alternatively: [gr])\n" #'revert-buffer)
```

Comment:
I thought you could remove this change `revert-buffer` does not work as `mu4e-update-mail-and-index` currently.